lnd_services.go

```diff
@@ -42,7 +42,7 @@ var (
 	minimalCompatibleVersion = &verrpc.Version{
 		AppMajor:  0,
 		AppMinor:  15,
-		AppPatch:  4,
+		AppPatch:  5,
```

Comment:
Do we need to bump the minimal compatible version here? It looks to me like we only need the compile time dependency for the output script types, but the RPC call should be available in earlier versions as well?

Reply:
Yes indeed, kept it at 0.15.4.